starter_kits/C/src/macros.c

```diff
@@ -0,0 +1,18 @@
+#define NORM_X(X)	((X) + map_size[0]) % map_size[0]
```

Comment:
These should be helper functions. As a macro it's also dangerously formatted - needs more () and so on. But there is almost no reason to use macros in modern C so let's not do that.